### PR TITLE
Update Pluggable Widgets 2 Code

### PR DESCRIPTION
**Pull request by ConnorLand**
This PR updates several code snippets in https://docs.mendix.com/howto/extensibility/create-a-pluggable-widget-two in order to align them with the most recent Mendix Widget Generator build.

### CODE REVIEW

File: content/howto/extensibility/create-a-pluggable-widget-two.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ Right now the input is editable for any user at all times. However, the input sh
 
 To add these restrictions, follow the instructions below:
 
-1. In *TextBox.xml* add the enumeration attribute for `Editable`: 
+1. In *TextBox.xml* add the enumeration attribute for `Editable` inside `propertyGroup` (where you put the attribute inside `propertyGroup` will affect how the attribute renders in the Mendix Studios):
```
Comment (Adam-Dupaski) [inline content/howto/extensibility/create-a-pluggable-widget-two.md]: Is "the Mendix Studios" (plural) necessary here? Is it verified that you will also see this in Mendix Studio (and not just Pro)?

File: content/howto/extensibility/create-a-pluggable-widget-two.md [hunk 2]
```diff
@@ -193,23 +193,30 @@ This section will teach you to add validation to your TextBox widget. Using micr
 	}
 	```
 
-	After altering this code, run `npm run build` to update the widget. Next, in Mendix Studio Pro, press <kbd>F4</kbd> to synchronize your project directory, and then right-click your TextBox widget and select **Update widget**. Then click **Run Locally**, and click **View** to see your changes. 
+Next, add `Fragment` to the current React import (shown below), and add a new `Alert` import underneath the existing imports in *TextBox.tsx*:
```
Comment (Adam-Dupaski) [inline content/howto/extensibility/create-a-pluggable-widget-two.md]: Why is this not step 5?

File: content/howto/extensibility/create-a-pluggable-widget-two.md [hunk 3]
```diff
@@ -193,23 +193,30 @@ This section will teach you to add validation to your TextBox widget. Using micr
...
 
+	```ts
+	import { Component, ReactNode, Fragment, createElement } from "react";
+	import { Alert } from "./components/Alert";
+	```
+	
+	After altering this code, run `npm run build` to update the widget. Next, in Mendix Studio Pro, press <kbd>F4</kbd> to synchronize your project directory, and then right-click your TextBox widget and select **Update widget**. Then click **Run Locally**, and click **View** to see your changes.
```
Comment (Adam-Dupaski) [inline content/howto/extensibility/create-a-pluggable-widget-two.md]: You have at least 4 steps in this paragraph. You should probably number them to make them clearer to the reader.

File: content/howto/extensibility/create-a-pluggable-widget-two.md (hunk 1 above)
Comment (ConnorLand) [reply content/howto/extensibility/create-a-pluggable-widget-two.md]: It is verified that this affects both, yes.

File: content/howto/extensibility/create-a-pluggable-widget-two.md [hunk 4]
```diff
@@ -193,23 +193,30 @@ This section will teach you to add validation to your TextBox widget. Using micr
 	}
 	```
 
-	After altering this code, run `npm run build` to update the widget. Next, in Mendix Studio Pro, press <kbd>F4</kbd> to synchronize your project directory, and then right-click your TextBox widget and select **Update widget**. Then click **Run Locally**, and click **View** to see your changes. 
+Next, add `Fragment` to the current React import (shown below), and add a new `Alert` import underneath the existing imports in *TextBox.tsx*:
 
+	```ts
```
Comment (Adam-Dupaski) [inline content/howto/extensibility/create-a-pluggable-widget-two.md]: I don't know what's going on with this code snippet, but it's turning everything below into code (look at the preview of this file). Please fix.

File: content/howto/extensibility/create-a-pluggable-widget-two.md (hunk 2 above)
Comment (ConnorLand) [reply content/howto/extensibility/create-a-pluggable-widget-two.md]: Changed to Step 5. This was attempted as a next item within Step 4, as it is within the same file. However my original language wasn't even tabbed in. 

Will favor using steps more for tasks going forward.

File: content/howto/extensibility/create-a-pluggable-widget-two.md (hunk 3 above)
Comment (ConnorLand) [reply content/howto/extensibility/create-a-pluggable-widget-two.md]: This paragraph, and all others like it, have been turned into sub-item lists.

File: content/howto/extensibility/create-a-pluggable-widget-two.md (hunk 4 above)
Comment (ConnorLand) [reply content/howto/extensibility/create-a-pluggable-widget-two.md]: Will check how this builds and respond accordingly.

**Pull request merged**